RecoJets/Configuration/python/RecoPFJets_cff.py

```diff
@@ -61,6 +61,9 @@
     )
 
 
+fixedGridRhoFastjetCentral = fixedGridRhoFastjetAll.clone(
+    maxRapidity = cms.double(2.5)
+    )
```

Comment:
it can stay open, just don't expect it to go far ;)
There are some requests building up for JetMET limited reprocessing in 74X, which may use this PR as well.
It's not clear to me still when and how that will be made (a branched-off special 74_1X release, probably)